fcrepo-http-api/src/test/java/org/fcrepo/integration/http/api/FedoraLdpIT.java

```diff
@@ -2856,6 +2856,41 @@ public void testGetObjectReferences() throws IOException {
         }
     }
 
+    @Test
+    public void testInboundReferencesFromBinary() throws Exception {
+        final Node referenceProp = NodeFactory.createURI("http://awoods.com/pointsAt");
+        final HttpPost postContainer = postObjMethod();
+        final String containerUri;
+        try (final CloseableHttpResponse response = execute(postContainer)) {
+            assertEquals(CREATED.getStatusCode(), getStatus(response));
+            containerUri = getLocation(response);
+        }
+
+        final HttpPost postBinary = postObjMethod();
+        postBinary.setHeader(CONTENT_TYPE, TEXT_PLAIN);
+        postBinary.setEntity(new StringEntity("Test text"));
+        final String binaryUri;
+        try (final CloseableHttpResponse response = execute(postBinary)) {
+            assertEquals(CREATED.getStatusCode(), getStatus(response));
+            binaryUri = getLocation(response);
+        }
+
+        final String linkRdf = "INSERT { <" + binaryUri + "> <http://awoods.com/pointsAt> <" + containerUri + "> . } " +
```

Comment:
Minor, but there is a helper method on the parent class of this test used for adding properties to resources which you could use here. `setProperty(binaryUri + "/" + FCR_METADATA, "http://awoods.com/pointsAt", "<" + containerUri + ">")`